### PR TITLE
update local function development to point to functions framework. Up…

### PR DESCRIPTION
**Pull request by jryoo**
…date documentation to say we are on Node.js 10 (from 6).

### CODE REVIEW

File: docs/sdk/app-development.md [hunk 1]
```diff
@@ -149,20 +149,53 @@ Once your Function has been deployed, you can check its status by running:
...
         $ sls status
 
-## Local testing
+## Running Locally
 
-You can test your function locally by doing the following:
+Running functions locally is extremely simple using Google's [functions-framework](https://cloud.google.com/functions/docs/functions-framework).
 
-        $ npm install -g @google-cloud/functions-emulator
+        $ npm install @google-cloud/functions-framework
```
Comment (duszans) [inline docs/sdk/app-development.md]: The same is repeated in line 160 so I think we can remove this line.

File: docs/sdk/app-development.md [hunk 2]
```diff
@@ -149,20 +149,53 @@ Once your Function has been deployed, you can check its status by running:
...
+
+        $ npm start
+        ...
+        Serving function...
+        Function: dronedeploy
+        URL: http://localhost:8080/
+
+You should be able to send requests to this function using curl from another terminal window. Note that you can generate an authentication token from the CLI which you can pass in as a bearer token to authenticate your function request.
+
+        $ sls getAuth
```
Comment (duszans) [inline docs/sdk/app-development.md:196]: I didn't remember we have this in the CLI - I assume it's a tested flow so I don't have anything to worry about :]

**Review by duszans — APPROVED** [review]
LGTM except one redundant line

File: docs/sdk/app-development.md (hunk 2 above)
Comment (jryoo) [reply docs/sdk/app-development.md:196]: yup! I've used it

File: docs/sdk/app-development.md (hunk 1 above)
Comment (jryoo) [reply docs/sdk/app-development.md]: thanks for the catch

**Pull request merged**